azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/tools/auth/AzureAuthManager.java

```diff
@@ -46,13 +46,13 @@
         AuthType authType = getAuthTypeFromString(auth.getType());
         Map<AuthType, ICredentialRetriever> retrieverMap = buildCredentialRetrievers(env);
```

Comment:
ChainedCredentialRetriever can be removed if you use `LinkedHashMap` which is ordered.